### PR TITLE
Fix - NotFound errors gracefully

### PR DESCRIPTION
**Pull request by dhingrak**
fixes: #285 

Before:
When the user clicks on ViewParkedMessages and the stream does not have any events or the stream does not exist
PersistentSubscriptionParkedStream_Before

After:
PersistentSubscriptionParkedStream_After

Before:
The deleted stream in a stream browser shows an alert popup with Not Found
DeletedStreamBefore

After: 
DeletedStream

### CODE REVIEW

**Review by thefringeninja — APPROVED** [review]

File: src/js/modules/competing/views/subscriptions.item.viewparkedmessages.tpl.html [hunk 1]
```diff
@@ -40,4 +40,12 @@ <h2 class="page-title">View Parked Messages - {{streamId}}::{{groupName}}</h2>
 			</td>
 		</tr>
 	</tbody>
+
+	<tbody ng-if="entries.length < 1">
+	<tr>
+		<td colspan="5">
+			<em>No parked events for current path: {{ $stateParams | json }}
```
Comment (hayley-jean) [inline src/js/modules/competing/views/subscriptions.item.viewparkedmessages.tpl.html:47]:
```suggestion
			<em>No parked events for subscription {{streamId}}::{{groupName}}
```

File: src/js/modules/streams/views/streams.item.events.tpl.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
 			</td>
 		</tr>
 	</tbody>
-	<tbody ng-hide="streams">
+	<tbody ng-if="streams.length < 1">
 		<tr>
 			<td colspan="5">
 				<em>No events for current path: {{ $stateParams | json }}
```
Comment (hayley-jean) [inline src/js/modules/streams/views/streams.item.events.tpl.html:55]:
```suggestion
				<em>No events for stream: {{ streamId }}
```

**Review by hayley-jean — CHANGES_REQUESTED** [review]
The changes look good, I've just made some suggestions to improve the messages for when events aren't found

File: src/js/modules/streams/views/streams.item.events.tpl.html (hunk 2 above)
Comment (dhingrak) [reply src/js/modules/streams/views/streams.item.events.tpl.html]: Changes Done

**Review by hayley-jean — APPROVED** [review]

**Pull request merged**